packages/quantification/src/quantification.ts

```diff
@@ -548,6 +471,36 @@ const getGrandfatheredTonneQuantities = ({
   };
 };
 
+/**
+ * Takes the year-over-year difference in soil carbon content across multiple polygons,
+ * sums the future and baseline values individually, then provides the average year-over-year
+ * difference over the first 10 years.
+ *
+ * @returns The average year-over-year change for the first 10 years after the switch year
+ */
+const getTenYearProjectedTonnesPerYear = ({
+  somscAnnualDifferencesBetweenFutureAndBaselineScenariosPerPolygon,
+}: {
+  somscAnnualDifferencesBetweenFutureAndBaselineScenariosPerPolygon: AnnualTotals[];
+}): number => {
+  return divide(
```

Comment:
I kinda feel like wrapping this whole thing in a `divide` makes it a little hard to track, could we like do all the complicated logic, store that and then average?

like:
```
const annualDifferences = <big map/reduce nonsense here>;
return divide(
  annualDifferences.sum(),
  10,
)
```

Reply:
Yep, can do